.github/workflows/build-and-push-instantclient-images.yml

```diff
@@ -57,25 +66,25 @@ jobs:
               fi
             fi
           else
-            echo "MANUAL: Rebuilding all"
-            ol='oraclelinux7 oraclelinux8'
-            ic="19 21"
+            echo "MANUAL: Rebuilding"
+            ol=$(echo "${{ github.event.inputs.ol || env.ol}}" | sed -e 's/[ ,]\+/ /g')
+            ic=$(echo "${{ github.event.inputs.ic || env.ic}}" | sed -e 's/[ ,]\+/ /g')
           fi
           echo "Rebuilding: ${ol} ${ic}"
           echo "::set-output name=ol::${ol}"
           echo "::set-output name=ic::${ic}"
           rm "${changes}"
 
       - name: Log into GitHub Container Registry
-        run: echo "${{ secrets.GHCR_TOKEN }}" | docker login ghcr.io -u ${{ secrets.GHCR_USER }} --password-stdin
+        run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${GITHUB_ACTOR,,} --password-stdin
 
       - name: Build Oracle Instant Client
         run: |
           for o in ${{ steps.linux-version.outputs.ol }}
           do
             for i in ${{ steps.linux-version.outputs.ic }}
             do
-              docker build --tag ghcr.io/oracle/${o}-instantclient:${i} OracleInstantClient/${o}/${i}
+              docker build --tag ghcr.io/${{ github.repository_owner }}/${o}-instantclient:${i} OracleInstantClient/${o}/${i}
```

Comment:
The `<OWNER>` in the repository URL must be lowercase, and `github.repository_owner` is case sensitive...